.infrastructure/04_secrets.tf

```
@@ -92,4 +92,7 @@ resource "aws_ssm_parameter" "cms_github_pat" {
   name  = "/cms/github_pat"
   type  = "SecureString"
   value = random_password.cms_github_pat.result
+  lifecycle {
+    ignore_changes = [value]
```

I would add a comment to motivate this configuration:

```suggestion
    ignore_changes = [
      # Ignore changes to value, because the value is updated manually
      value
    ]
```